authentik/sources/oauth/api/property_mapping.py

```diff
@@ -0,0 +1,39 @@
+"""OAuth source property mappings API"""
+
+from django_filters.filters import AllValuesMultipleFilter
+from django_filters.filterset import FilterSet
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import extend_schema_field
+from rest_framework.viewsets import ModelViewSet
+
+from authentik.core.api.propertymappings import PropertyMappingSerializer
+from authentik.core.api.used_by import UsedByMixin
+from authentik.sources.oauth.models import OAuthSourcePropertyMapping
+
+
+class OAuthSourcePropertyMappingSerializer(PropertyMappingSerializer):
+    """OAuthSourcePropertyMapping Serializer"""
+
+    class Meta:
+        model = OAuthSourcePropertyMapping
+        fields = PropertyMappingSerializer.Meta.fields
+
+
+class OAuthSourcePropertyMappingFilter(FilterSet):
```

Comment:
we can probably have a base class for this to inherit from too, if its just for that managed field